package.json

```diff
@@ -105,7 +105,7 @@
     "generator-rn-toolbox": "^3.11.1",
     "get-dev-paths": "^0.1.1",
     "http-proxy": "^1.17.0",
-    "jest": "^24.9.0",
+    "jest": "^25.1.0",
```

Comment:
The last time I checked there were no jest types for v25 yet. I think it is best to stick with 24 until they are available.

Reply:
Or is this recommended by the diff? In this case it is still weird if there are no flow types.

Reply:
Yep, good catch :+1: